core/modules/layout/includes/block.text.inc

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ *  TextBlock extends Block
+ *  
+ *  This class allows us to create text blocks.
+ */
+class TextBlock extends Block {
+  /**
+   *  Sets title text on draggable block panel in Layout builder.
+   */
+  function getAdminTitle() {
+    return t('Text block');
```

Comment:
Let's move the display label up into the admin title (`$this->settings['subject']`). For reusable blocks (once we have those) we can use the admin label (`info`) instead.